inception-project/inception
# #4058 - Cannot export texts containing certain characters as UIMA CAS XMI

**What's in the PR**
- Rename packages in XMI support module
- Switch to modern spring factories declaration
- Enable sanitziing of illegal characters on export for XMI formats
- Added option to control if sanitation happens or not
- Updated documentation of XMI formats

**How to test manually**
* Create a small ZIP file
* Import it as a plain text document
* Export then project using the secondary format `XMI (XML 1.0)` or `XMI (XML 1.1)` (should work)
* Add the line `format.uima-xmi.sanitize-illegal-characters=false` to the `settings.properties` file
* Export then project using the secondary format `XMI (XML 1.0)` (should fail)

**Automatic testing**

**Documentation**
